pkgs/tools/security/sequoia/default.nix

```diff
@@ -25,16 +25,16 @@ rustPlatform.buildRustPackage rec {
   pname = "sequoia";
```

Comment:
Please remove glib from inputs and ``? null`` with the assert.

Reply:
I'll push more commits including the one fixing this, after I'll get a comment from upstream at https://gitlab.com/sequoia-pgp/sequoia/-/merge_requests/1097 .